stable_baselines3/common/torch_layers.py

```diff
@@ -98,6 +98,7 @@ def create_mlp(
     output_dim: int,
     net_arch: List[int],
     activation_fn: Type[nn.Module] = nn.ReLU,
+    with_bias: bool = True,
```

Comment:
minor: when adding parameter, I usually prefer to add them at the end (otherwise, it may break some code when not called as keyword argument)

Reply:
Indeed, it's safer